pkgs/tools/misc/routino/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchsvn, perl, zlib, bzip2 }:
+
+stdenv.mkDerivation rec {
+  name = "Routino-${version}";
+  version = "3.2";
+
+  src = fetchsvn {
+    url = "http://routino.org/svn/trunk/";
+    rev = 1903;
+    sha256 = "0brrybd9jna47wbsxbdx2f21z1jl602n6686071fj69lmghv7ilz";
+  };
+
+  nativeBuildInputs = [ perl ];
+  buildInputs = [ zlib bzip2 ];
+
+  installPhase = ''
+    make DESTDIR="$out" install
+    mv $out/usr/local/* $out
+    rmdir $out/usr/local $out/usr
+  '';
```

Comment:
Replace this with `makeFlags = [ "prefix=$out" ];`.

Reply:
I could indeed do `installFlags = [ "DESTDIR=$out" ];`.
But nevertheless, the install target creates a directory `$out/usr/local` from which I can move the content to `$out` in the `postInstall` phase.

Reply:
I'm just remebering that this doesn't work either:
I intentionally wrote `make DESTDIR="$out" install` since if I don't quote the variable, the package is installed to a directory called `ut/`.
I tried every way of escaping/quoting I was able to come up with:
- `installFlags = [ ''DESTDIR="$out"'' ]`
- `installFlags = [ "DESTDIR=\$out" ]`
- ...

but the package is always installed to `ut/`.
Do you have an idea?

Reply:
No, that `/usr/local` comes from the default `prefix` in the Makefile. If you define it with `makeFlags = [ "prefix=$out" ];`, you will not have to override `DESTDIR`.

Reply:
Forget about it, I found a solution:
`installFlags = [ "DESTDIR=$(out)" ];`

Reply:
You're absolutely right, but `makeFlags = [ "prefix=$out" ];` won't work.
For some strange reason, you need `"prefix=$(out)"`.

Reply:
Sorry, `prefix` indeed is not overridable with a make flag. I would try `substituteInPlace Makefile.conf --replace prefix=/usr/local prefix=$out`in the `postPatch` phase.